lib/publish.js

```diff
@@ -27,6 +28,8 @@ module.exports = async (pluginConfig, context) => {
     baseRef,
   } = resolveConfig(pluginConfig, context);
   const { owner, repo } = parseGithubUrl(repositoryUrl);
+  const absoluteRepoPath = (await execa('git', ['rev-parse', '--show-toplevel'])).stdout;
```

Comment:
I know this plugin is made to work on git repos, but what if it actually not run on a git repo? This command might fail. In that case, the fallback value for `absoluteRepoPath` can be `./`. Shouldn't we do a try/catch to make it happen?

Reply:
Sure sounds good. :+1:

Reply:
Fixed up now.